pygeo/constraints/areaConstraint.py

```diff
@@ -68,6 +68,8 @@ def __init__(
         self.perim_scale = perim_scale
         self.max_perim = max_perim
         self.smSize = None
+        self.perim_length = None
```

Comment:
Is this to avoid attribute injection at a later stage?

Reply:
This attribute is injected somewhere (not in the init), so I've defined it here to make class attributes more explicit.